lib/linguist/languages.yml

```diff
@@ -5897,6 +5878,35 @@ Windows Registry Entries:
   codemirror_mode: properties
   codemirror_mime_type: text/x-properties
   language_id: 969674868
+Wolfram Language:
+  type: programming
+  extensions:
+  - ".cdf"
+  - ".m"
+  - ".ma"
+  - ".mt"
+  - ".nb"
+  - ".nbp"
+  - ".wl"
+  - ".wls"
+  - ".wlt"
+  filenames:
+  - init.m
+  - init.wl
```

Comment:
The `.m` and `.wl` file extensions are already associated with Wolfram Language, so adding these filenames won't do anything.